DEV/org.openl.core/src/org/openl/binding/impl/MethodNodeBinder.java

```diff
@@ -121,6 +124,19 @@ public IBoundNode bind(ISyntaxNode node, IBindingContext bindingContext) throws
         }
     }
 
+    private void validateMethodParameter(IMethodCaller methodCaller, IBoundNode[] children) {
+        if (methodCaller instanceof CastingMethodCaller) {
```

Comment:
`methodCaller.getMethod().getSignature().getParameterTypes()` returns method argument types, which can be used for validation without casting and calling runtime functionality.
Validation can be processed via `org.openl.rules.binding.RuleRowHelper#validateValue`

Reply:
Be aware for [MultiCallOpenMethod](https://github.com/openl-tablets/openl-tablets/blob/master/DEV/org.openl.core/src/org/openl/binding/impl/method/MultiCallOpenMethod.java) implementation.